### DISCUSSION TITLE
PowerShell not recognized 'starship' term

### DISCUSSION THREAD

**Discussion by OsirisFrik** [starship/starship#2071] [Q&A] [answered]
Hi, I try to run starship on powerShell, but don't work and return me this error

```
 The term 'starship' is not recognized as the name of a cmdlet, function, script file, or operable program.
```

![image](https://user-images.githubusercontent.com/9331532/103569333-f5665380-4e7b-11eb-9862-90f86e2c4f79.png)

**Comment by matchai** [comment, marked as answer]
Hi @OsirisFrik,

Which installation method did you use to install Starship?
This seems like it would be an issue with your package manager (scoop or chocolatey) rather than Starship itself.

**Reply by OsirisFrik** [reply]
I used the first option with git-bash

```sh
curl -fsSL https://starship.rs/install.sh | bash
```

ok, I try with chocolatey and work, tnks
